test/test-init.sh

```diff
@@ -30,7 +30,8 @@ begin_test "init with old settings"
 
   [ "$res" = 2 ]
 
-  grep "clean attribute should be" init.log
+  cat init.log
```

Comment:
It only shows up if the test fails. It's much easier to diagnose with this printed out.